src/openrct2/Limits.h

```diff
@@ -8,16 +8,19 @@
  *****************************************************************************/
 
 #pragma once
+
 #include "rct2/Limits.h"
 
+#include <limits>
+
 namespace OpenRCT2::Limits
 {
     constexpr uint16_t kMaxRidesInPark = 1000;
     constexpr uint16_t kMaxStationsPerRide = 255;
     constexpr uint8_t kCustomerHistorySize = RCT12::Limits::kCustomerHistorySize;
     constexpr uint16_t kMaxGolfHoles = RCT12::Limits::kMaxGolfHoles;
     constexpr uint16_t kMaxHelices = RCT12::Limits::kMaxHelices;
-    constexpr uint16_t kMaxInversions = RCT12::Limits::kMaxInversions;
```

Comment:
Isn't `RCT12::Limits::kMaxInversions` now unused?

Reply:
Probably yes. Not sure if we should remove it, though.

Reply:
I commented them out on 48073edb7e6cd1287a0a67e744368517c5cb3e47